src/views/FeedList.vue

```diff
@@ -87,6 +131,17 @@ export default {
     }
   }
 
+  .feed-category-row {
+    background-color: $light-color;
+    padding: 0.5rem 1rem;
+
+    .filter-button {
+      svg {
+        height: 0.8rem;
+      }
```

Comment:
```suggestion
      }
      
      span {
        vertical-align: middle;
      }
```
and to remove 'vertical-align-mid' from the above, because it is the same as inlined styles